esrally/reporter.py

```diff
@@ -301,6 +303,33 @@ def _report_ingest_pipeline_stats(self, stats):
             self._line("Total Ingest Pipeline failed", "", stats.ingest_pipeline_cluster_failed, ""),
         )
 
+    def _report_field_disk_usage(self, stats):
+        # Collect stats so we can easily make lines
+        collated = {}
+        for stat, fieldStats in {
+            "inverted index": stats.field_disk_usage_inverted_index,
+            "stored fields": stats.field_disk_usage_stored_fields,
+            "doc values": stats.field_disk_usage_doc_values,
+            "points": stats.field_disk_usage_points,
+            "norms": stats.field_disk_usage_norms,
+            "term vectors": stats.field_disk_usage_term_vectors,
+            "total": stats.field_disk_usage_totals,
+        }.items():
+            for fieldStat in fieldStats:
+                collated.setdefault(fieldStat["index"], {}).setdefault(fieldStat["field"], {})[stat] = fieldStat["value"]
+
+        # Sort by total disk usage descending
```

Comment:
Another option is to sort by field name and index name alphabetically. But for my usage getting the biggest fields below the smallest fields is nice.